library-model-loader/src/main/java/edu/ucr/cs/riple/librarymodel/LibraryModelLoader.java

```diff
@@ -38,21 +41,28 @@
 public class LibraryModelLoader implements LibraryModels {
 
   public final String NULLABLE_METHOD_LIST_FILE_NAME = "nullable-methods.tsv";
+  public final String NULLABLE_FIELD_LIST_FILE_NAME = "nullable-fields.tsv";
   public final ImmutableSet<MethodRef> nullableMethods;
+  public final ImmutableSet<FieldRef> nullableFields;
 
   // Assuming this constructor will be called when picked by service loader
   public LibraryModelLoader() {
-    this.nullableMethods = parseTSVFileFromResourcesToMethodRef(NULLABLE_METHOD_LIST_FILE_NAME);
+    this.nullableMethods =
+        parseTSVFileFromResourcesToMemberRef(
+            NULLABLE_METHOD_LIST_FILE_NAME, values -> methodRef(values[0], values[1]));
+    this.nullableFields =
+        parseTSVFileFromResourcesToMemberRef(
+            NULLABLE_FIELD_LIST_FILE_NAME, values -> fieldRef(values[0], values[1]));
   }
 
   /**
-   * Loads a file from resources and parses the content into set of {@link
-   * com.uber.nullaway.LibraryModels.MethodRef}.
+   * Loads a file from resources and creates an instance of type T from each line of the file.
    *
    * @param name File name in resources.
-   * @return ImmutableSet of content in the passed file. Returns empty if the file does not exist.
+   * @return ImmutableSet of contents in the file. Returns empty if the file does not exist.
    */
-  private ImmutableSet<MethodRef> parseTSVFileFromResourcesToMethodRef(String name) {
+  private <T> ImmutableSet<T> parseTSVFileFromResourcesToMemberRef(
```

Comment:
Not for this PR, but, if in NullAway we had a common marker interface `MemberRef` for `MethodRef` and `FieldRef`, then `MemberRef` could be an upper bound for `T`, giving slightly more type safety

Reply:
Sounds good. Added the issue #222.